src/plugins/intel_gpu/include/intel_gpu/graph/kernel_impl_params.hpp

```diff
@@ -53,6 +54,7 @@ struct kernel_impl_params {
     size_t primary_input_idx = 0;
 
     memory::ptr reordered_weights = nullptr;
+    network *net = nullptr;
```

Comment:
network::ptr can be used.

Reply:
Or isn't it enough to have stream::ptr?

Reply:
Yes, like 2nd option, `stream::ptr` is just enough. updated as your suggestion.